pkg/scheduler/preemption/preemption.go

```diff
@@ -386,8 +386,11 @@ func (p *Preemptor) fairPreemptions(preemptionCtx *preemptionCtx, strategies []f
 	if len(candidates) == 0 {
 		return nil
 	}
-	sort.Slice(candidates, func(i, j int) bool {
-		return CandidatesOrdering(preemptionCtx.log, p.enabledAfs, candidates[i], candidates[j], preemptionCtx.preemptorCQ.Name, p.clock.Now())
+	slices.SortFunc(candidates, func(a, b *workload.Info) int {
+		if CandidatesOrdering(preemptionCtx.log, p.enabledAfs, a, b, preemptionCtx.preemptorCQ.Name, p.clock.Now()) {
```

Comment:
We should probably refactor CandidatesOrdering to return an int. However, I think we can address this in a follow-up PR.

Reply:
I think that refactoring `CandidatesOrdering` is the scope of this PR.